projects/ngneat/helipopper/src/lib/utils.ts

```diff
@@ -28,18 +30,20 @@ export function inView(
       return;
     }
 
-    const observer = new IntersectionObserver(entries => {
-      // Several changes may occur in the same tick, we want to check the latest entry state.
-      const entry = entries[entries.length - 1];
-      if (entry.isIntersecting) {
-        subscriber.next();
-        subscriber.complete();
-      }
-    }, options);
+    return ngZone.runOutsideAngular(() => {
```

Comment:
I was pretty sure I already run everything outside the zone. I guess I was wrong. Thanks!